dtos/address.go

```diff
@@ -103,6 +117,20 @@ func NewEmailAddress(recipients []string) Address {
 	}
 }
 
+type MessageBus struct {
+	Topic string `json:"topic,omitempty" validate:"required"`
+}
+
+type ZeroMQPubAddress struct {
+}
+
+func NewZeroMQPubAddress(topic string) Address {
+	return Address{
+		Type:       common.EMAIL,
+		MessageBus: MessageBus{Topic: topic},
+	}
+}
```

Comment:
this snippet seems incomplete

Reply:
Change the wrong type from EMAIL to ZeroMQ.

Reply:
The ZeroMQPubAddress DTO only need the topic field and Base Address's Type, Host, Port. So it is an empty struct to consist with other Address DTO.